t/t5318-commit-graph.sh

```diff
@@ -116,6 +116,24 @@ test_expect_success 'Add more commits' '
 	git repack
 '
 
+test_expect_success 'commit-graph write displays progress by default i.e. without any progress option' '
+	cd "$TRASH_DIRECTORY/full" &&
+	git commit-graph write 2>progress &&
+	test_file_not_empty progress
+'
+
+test_expect_success 'commit-graph write displays progress with the --progress option' '
```

Comment:
we talked offline that this case could be dropped, as it is enough that the default has progress and `--no-progress` does not have progress.